<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>169</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="options_layout">
     <item row="0" column="0">
      <widget class="QLabel" name="function_label">
       <property name="text">
        <string>Function</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="function_comboBox"/>
     </item>
     <item row="1" column="1">
      <layout class="QHBoxLayout" name="width_layout">
       <item>
        <widget class="QSlider" name="width_horizontalSlider">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>25</number>
         </property>
         <property name="pageStep">
          <number>1</number>
         </property>
         <property name="value">
          <number>2</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="invertedAppearance">
          <bool>false</bool>
         </property>
         <property name="invertedControls">
          <bool>false</bool>
         </property>
         <property name="tickPosition">
          <enum>QSlider::TicksBelow</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="width_spinBox"/>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="width_label">
       <property name="text">
        <string>Smooth Width</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="blend_groupBox">
     <property name="title">
      <string>Blend Selection Edges</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QFormLayout" name="blendWidth_formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="blendWidth_label">
          <property name="text">
           <string>Blend Width</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <layout class="QHBoxLayout" name="blendWidth_horizontalLayout">
          <item>
           <widget class="QSlider" name="blendWidth_horizontalSlider">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>25</number>
            </property>
            <property name="pageStep">
             <number>1</number>
            </property>
            <property name="value">
             <number>2</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="invertedAppearance">
             <bool>false</bool>
            </property>
            <property name="invertedControls">
             <bool>false</bool>
            </property>
            <property name="tickPosition">
             <enum>QSlider::TicksBelow</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="blendWidth_spinBox"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
